package.json

```diff
@@ -6,6 +6,8 @@
     "start": "next start"
   },
   "dependencies": {
+    "@tanstack/react-query": "^5.51.23",
+    "axios": "^1.7.3",
```

Comment:
I think you did not use this package (`axios`) in your code because you are using `fetch`. You can remove it.